index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!--this is header section-->
+    <header class="header-container">
+        <h2 class="welcome">Welcome</h2>
+        <a class="menu-btn"><img src="/assets/images/Union.png" alt="menu button"></a>
+    </header>
+    <!--end of header section-->
+
+    <!--the main section-->
+    <main class="main-section-container">
+        <h2 class="main-title main-section-title">Hey there. I'm</h2>
+        <h2 class="main-title main-section-title">Maen</h2>    
```

Comment:
Please ensure that the container of this section is a `section` tag. You can have something like this:

`<main>`

      <section>
             headline section
      </section>

`</main>`